### DISCUSSION TITLE
Column component. trouble with the rendering of json items

### DISCUSSION THREAD

**Discussion by SebastiendOrnano** [sqlpage/SQLPage#737] [Q&A]
I am exploring the column component but the json items are rendered as text.

So I copy paste the first example of the documentation in the index.sql file on clean install (sqlpage.bin 0.32)but with de same  result
![Column](https://github.com/user-attachments/assets/353dd165-bba4-47f7-8795-d6ea662cc3cb)

The inspection give nothing
![Inspect](https://github.com/user-attachments/assets/43c649a7-7067-47d1-a418-d1c658febe4b)

I am working on linux mint 22, firefox 133 (I did a test after disabling all add_ons)

So, I can't understand where the problem is coming from.

Any help is  welcome and thanks in advance

**Comment by lovasoa** [comment]
Hello ! Can you follow step by step instructions to reproduce the issue ?

See the instructions I sent last time : https://github.com/sqlpage/SQLPage/discussions/712#discussioncomment-11373147

When reporting an issue, the best way for you to debug it and for us to work on it then is to reproduce in a clean environment, including starting a new project and downloading a sqlpage binary.

**Reply by SebastiendOrnano** [reply]
Same result
![test](https://github.com/user-attachments/assets/32acd8a1-c597-42c6-8ba5-72b47b6176d0)

I did try with a clean new install on another computer (linux mint 21.3 firefox with no add on and with chromium) with the same result

**Reply by lovasoa** [reply]
Indeed! I can reproduce it too. Sorry and thank you. I'll look into it.

In the mean time, you can work around the issue by using the json_object syntax:

```sql
    JSON_OBJECT('icon','database', 'color','blue', 'description','1GB Database') as item,
```

**Comment by lovasoa** [comment]
It's fixed; will be released in 0.32. Thanks again @SebastiendOrnano : https://github.com/sqlpage/SQLPage/discussions/737
